src/olympia/bandwagon/tests/test_views.py

```diff
@@ -1337,6 +1340,31 @@ def test_sort(self):
         assert response.data['results'][1]['uuid'] == col_a.uuid
         assert response.data['results'][2]['uuid'] == col_c.uuid
 
+    def test_has_addon(self):
+        col_a = collection_factory(author=self.user)
+        col_b = collection_factory(author=self.user)
+        col_c = collection_factory(author=self.user)
+        col_a.update(modified=self.days_ago(3))
+        col_b.update(modified=self.days_ago(1))
+        col_c.update(modified=self.days_ago(6))
+        collection_factory(author=user_factory())  # Not our collection.
+        Collection.objects.all().count() == 4
+
+        self.client.login_api(self.user)
+        response = self.client.get(self.url, {'has_addon': 42})
+        assert response.status_code == 200
+        assert len(response.data['results']) == 3
+        assert response.data['results'][0]['has_addon'] is False
+        assert response.data['results'][1]['has_addon'] is False
+        assert response.data['results'][2]['has_addon'] is False
+
```

Comment:
can we have a test where `has_addon` is True?

Reply:
Ugh I thought I did... Let me add that

Reply:
Done in https://github.com/mozilla/addons-server/pull/7222/commits/083127e9746935711964e1312057351438e8c263